plugins/webp-uploads/tests/test-image-edit.php

```diff
@@ -47,18 +47,12 @@ public function test_it_should_backup_the_sources_structure_alongside_the_full_s
 		$this->assertArrayHasKey( 'full-orig', $backup_sources );
 		$this->assertSame( $metadata['sources'], $backup_sources['full-orig'] );
 
+		unset( $backup_sizes['full-orig'] );
 		foreach ( $backup_sizes as $size => $properties ) {
 			$size_name = str_replace( '-orig', '', $size );
-
-			if ( 'full-orig' === $size ) {
-				continue;
-			}
-
 			$this->assertArrayHasKey( 'sources', $properties );
 			$this->assertSame( $metadata['sizes'][ $size_name ]['sources'], $properties['sources'] );
 		}
-
-		$metadata = wp_get_attachment_metadata( $attachment_id );
```

Comment:
This appears to be dead code, perhaps code which was going to have assertions added to it, but they never were written.